backend/src/nodes/image_util_nodes.py

```diff
@@ -414,6 +423,8 @@ def __init__(self):
     def run(
         self, img: np.ndarray, angle: float, interpolation: int, expand: int, fill: int
     ) -> np.ndarray:
+        """Rotates an image by the desired angle."""
```

Comment:
Why? This doc string adds no information. I mean, what else is the "RotateNode" with the description "Rotate an image." going to do?

I think is a general problem in the python code base. We have a bunch useless/trivial doc strings. I mean, what value does a "Constructor" doc string inside `__init__` have?

Reply:
We should get rid of all those tbh. It's from the initial class I had set up with generic docstrings, and they really do nothing but clog up all the files now

Reply:
I only added it because the inconsistency bothered me haha